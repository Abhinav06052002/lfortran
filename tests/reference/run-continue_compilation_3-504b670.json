{
    "basename": "run-continue_compilation_3-504b670",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_3.f90",
    "infile_hash": "2bf81cf1328e6352facfffe9604bf8413a6448d926ed53e523594cc5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-continue_compilation_3-504b670.stdout",
    "stdout_hash": "81a81e2bff825f09ef9e139c3e0164ab70cecd89dd83428e57d2268c",
    "stderr": "run-continue_compilation_3-504b670.stderr",
    "stderr_hash": "c56db9b45a76f39c6c3e7d457d82d59c0d5bd24b5fc5cdb90939f5b5",
    "returncode": 1
}