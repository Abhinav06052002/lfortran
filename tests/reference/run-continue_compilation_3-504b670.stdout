(TranslationUnit
    (SymbolTable
        1
        {
            continue_compilation_3:
                (Program
                    (SymbolTable
                        2
                        {
                            1_t_xx:
                                (ExternalSymbol
                                    2
                                    1_t_xx
                                    3 xx
                                    t
                                    []
                                    xx
                                    Public
                                ),
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 5 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a1:
                                (Variable
                                    2
                                    a1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b:
                                (Variable
                                    2
                                    b
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 10 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            bpe:
                                (Function
                                    (SymbolTable
                                        4
                                        {
                                            
                                        })
                                    bpe
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    []
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            c:
                                (Variable
                                    2
                                    c
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            k:
                                (Variable
                                    2
                                    k
                                    []
                                    Local
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    Save
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            s:
                                (Variable
                                    2
                                    s
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (String 1 8 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            s1:
                                (Variable
                                    2
                                    s1
                                    []
                                    Local
                                    (StringConstant
                                        "lfortran"
                                        (String 1 8 () PointerString)
                                    )
                                    (StringConstant
                                        "lfortran"
                                        (String 1 8 () PointerString)
                                    )
                                    Save
                                    (String 1 8 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            str:
                                (Variable
                                    2
                                    str
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (String 1 4 () PointerString)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            t:
                                (Struct
                                    (SymbolTable
                                        3
                                        {
                                            xx:
                                                (Variable
                                                    3
                                                    xx
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Integer 4)
                                                    ()
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    t
                                    []
                                    [xx]
                                    Source
                                    Public
                                    .false.
                                    .false.
                                    []
                                    ()
                                    ()
                                ),
                            x:
                                (Variable
                                    2
                                    x
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            x1:
                                (Variable
                                    2
                                    x1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 2 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            x3:
                                (Variable
                                    2
                                    x3
                                    []
                                    Local
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y:
                                (Variable
                                    2
                                    y
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (StructType
                                        2 t
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    continue_compilation_3
                    []
                    [(Assignment
                        (Var 2 a)
                        (Var 2 a)
                        ()
                    )
                    (Assignment
                        (Var 2 i)
                        (IntegerUnaryMinus
                            (IntegerConstant 3 (Integer 4) Decimal)
                            (Integer 4)
                            (IntegerConstant -3 (Integer 4) Decimal)
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerConstant 0 (Integer 4) Decimal)
                        ()
                    )
                    (GoToTarget
                        1
                        1
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 1 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (GoToTarget
                        2
                        2
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 2 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (GoToTarget
                        3
                        3
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 4 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 c)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )
                    (If
                        (IntegerCompare
                            (Var 2 c)
                            NotEq
                            (IntegerConstant 7 (Integer 4) Decimal)
                            (Logical 4)
                            ()
                        )
                        [(ErrorStop
                            ()
                        )]
                        []
                    )
                    (Assignment
                        (Var 2 i)
                        (Var 2 i)
                        ()
                    )
                    (Assignment
                        (Var 2 i)
                        (Var 2 i)
                        ()
                    )
                    (Assignment
                        (Var 2 x3)
                        (IntegerConstant 1 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 x3)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )
                    (Assignment
                        (Var 2 i)
                        (Var 2 i)
                        ()
                    )
                    (Assignment
                        (Var 2 s)
                        (StringConstant
                            "lfortran"
                            (String 1 8 () PointerString)
                        )
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(StringConstant
                                "Length of s:"
                                (String 1 12 () PointerString)
                            )
                            (StringLen
                                (Var 2 s)
                                (Integer 4)
                                ()
                            )]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )
                    (SubroutineCall
                        2 bpe
                        ()
                        []
                        ()
                    )
                    (Assignment
                        (Var 2 i)
                        (Var 2 i)
                        ()
                    )]
                )
        })
    []
)
